test_util/src/lib.rs

```diff
@@ -2198,9 +2198,12 @@ pub fn parse_strace_output(output: &str) -> HashMap<String, StraceOutput> {
   // Filter out non-relevant lines. See the error log at
   // https://github.com/denoland/deno/pull/3715/checks?check_run_id=397365887
   // This is checked in testdata/strace_summary2.out
-  let mut lines = output
-    .lines()
-    .filter(|line| !line.is_empty() && !line.contains("detached ..."));
+  let mut lines = output.lines().filter(|line| {
+    !line.is_empty()
+      && !line.contains("detached ...")
+      && !line.contains("unfinished ...")
+      && !line.contains("????")
```

Comment:
This line is future proofing, since I do not know all possible strace states that are sometimes present at the beginning of the file, but it seems that all of them contains `????`

```
87381 ????( <unfinished ...>
87358 ????( <detached ...>
```